reverse_script.py

```diff
@@ -0,0 +1,64 @@
+import os, sys
+import re
+import json
+
+# DESCRIPTION
+# We are launching script from the command line 
+# Taking the name of the file with the dictionary (json) from the command
+# Reversing dictionary
+# Writing the result to the file (the name of the file present in system variable)
+
+# reverting dictionary (changing keys with values)
+def reversing_dict(dictionary):
+    result_dict = {}
+    buffer = []
+    for key, value in dictionary.items():
+        if value not in result_dict.keys(): #adding new entry if it is a new key
+            result_dict[value] = key
+        else:   #adding new value to the existed one to the same key
+            buffer = list(result_dict[value])
+            buffer.append(key)
+            result_dict[value] = buffer
+    return result_dict
+
+# set up a path to the reseult file as constanta 
+# (to allow launch it on any Win PC)
+RESULT_FILE_PATH = os.environ["TEMP"] + '\\resulted_dictionary.json'
+
+# checking if such environmental variable is present is the system
+# writing needed path to it if yes
+# adding a new entry to the mapping if not
+if 'REVDICTPATH' in os.environ:
+    file_path_env = os.environ['REVDICTPATH']
+else:
+    os.environ['REVDICTPATH'] = RESULT_FILE_PATH
+
+try:
+    # reading command
+    source_file_path = sys.argv[1]

```

Comment:
JFYI For senior level, it is good to be aware of 'argparse' standard library (for example, it is using to add own CLI(command-line-interface) options to pytest testing framework)
Also, 'configparser' (for reading config from sections of INI files) is often used.

Reply:
fixed in f6c1ecd3e51d763fcd7446c3ac8abefabefafea2